specification.md

```diff
@@ -2119,6 +2146,38 @@ message TLSConnectionSettings {
 }
 ```
 
+#### ProxyConnectionSettings Message
+
+Status: [Development]
+
+The message carries optional proxy settings that are used to configure a
+client's connection. If the Agent is able to validate the connection settings,
+the Agent SHOULD forget any previous proxy settings. If this message is not
+included, the client SHOULD use the agent's default proxy settings for the
+connection.
```

Comment:
Should we use the same language from your update to the TLS settings?

> client SHOULD assume the settings are unchanged and continue using existing
settings.